### DISCUSSION TITLE
Artwork besides cover

### DISCUSSION THREAD

**Discussion by ITguy3118** [beetbox/beets#4284] [Ideas]
It would be nice to see other artworks besides just the cover pulled in, also the spin, back, booklet, disc image. These would be optional artworks to grab of course, but I like to have complete items of artwork to pull in for my use of the album within the kodi platform. Please find a source/way to implement this.

**Comment by wisp3rwind** [comment]
> Please find a source/way to implement this.

Contributions would be very welcome :)

For context, this probably depends on someone (...) reviving the ["Attachment Proposal"](https://github.com/beetbox/beets/wiki/Attachment-Proposal).

EDIT: Also, the idea is not new, there must be several issues requesting & discussing this feature already.
